src/Compilers/CSharp/Test/Syntax/Diagnostics/LineSpanDirectiveTests.cs

```diff
@@ -545,15 +550,18 @@ static void Main()
         B();
 #line (1, 1) - (1, 100) ""b.txt""
         C();
+#line (2, 3) - (2, 4) 9 ""a.txt""
+        D();
     }
     static void A() { }
     static void B() { }
     static void C() { }
+    static void D() { }
```

Comment:
Does this cause the test to exhibit the bug fixed in this PR?

Reply:
No, this test change is unrelated to the bug fix, and is only here to test a more interesting case than the line above.